code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -188,7 +188,7 @@ Pipelines + Other Objects -> Pipe network
 
 /obj/machinery/atmospherics/wrench_act(mob/living/user, obj/item/I)
 	if(!can_unwrench(user))
-		return TRUE
+		return ..()

```

Comment:
Shouldn't this be if they -can- wrench it?

Reply:
Can unwrench is used solely by pipes, connectors and what not but not machines. It's a snowflake overwrite, but not doing that return there will block wrenches from doing anything ie:can't hit things with a wrench or other actions